src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -22,7 +25,25 @@ public class AddCommandIntegrationTest {
 
     @BeforeEach
     public void setUp() {
-        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        model = new ModelManager(getTypicalPennyWise(), new UserPrefs());
+    }
+
+    @Test
+    public void execute_newExpenditure_success() {
+        Entry validExpenditure = new ExpenditureBuilder().build();
+        Model expectedModel = new ModelManager(model.getPennyWise(), new UserPrefs());
+        expectedModel.addExpenditure(validExpenditure);
+        assertCommandSuccess(new AddCommand(validExpenditure, new EntryType("e")), model,
```

Comment:
```suggestion
        assertCommandSuccess(new AddCommand(validExpenditure, new EntryType(EntryType. ENTRY_TYPE_EXPENDITURE)), model,
```

I think it's better if we use the defined constants

Reply:
good point!